documentation/data-importer.adoc

```diff
@@ -0,0 +1,341 @@
+But you should keep in mind that it is only a preview and only a sample of your data.
+The actual graph may look different once the full import is complete.
+
+You want to have your mapping completely done before previewing, but if you have missed something, you are still able to do a preview, but the incomplete elements will not be rendered.
+In other words, any dashed element in your model will not show up in the preview.
+
+As mentioned before, if any element in your model is dashed, you will **not** be able to run the import.
+
+== Running an import
+
+You have drawn your data model, mapped every single element, and checked the preview to make sure things connect the way you intended.
+That means you are ready to run the actual import.
+
+Click _Run import_ to import the files specified in your model. 
+The simple model you have created in this example only maps to three of the eleven available files.
+The files not specified in your model are **not** imported, but they remain in the _Files_ section on your left.
+
+If any mapping is incomplete, Data Importer will tell you that your model has errors and add a red exclamations mark next to the incomplete element in the model.
+When you select an incomplete element, the mapping pane also highlights the missing details.
+
+Once you've addressed the errors, go ahead and run the import again.
+You should see a popup window showing the successful results.
+
+image::import_results.png[]
+
+The _Import results_ summarizes what was included in the import. 
+You can compare to your CSV files to verify that everything was imported.
+
+== Cypher statements
+
+You can also see the Cypher commands used to create the elements in your model.
+Let's have a look at _Order_ nodes.
+Click _Show Cypher_ on _Order_.
+The key statement:
+
+.Key statement
+[source,cypher]
+----
+CREATE CONSTRAINT `imp_uniq_Order_orderID` IF NOT EXISTS
+FOR (n: `Order`)
+REQUIRE (n.`orderID`) IS UNIQUE;
+----
+
+This ensures that the `orderID` is unique which is important since this is what is used later to create the relationship between the _Order_ nodes and the _Product nodes.
+
+The load statement creates all the different nodes sets all your selected properties.
+Both key and load statements look similar to this for the _Product_ nodes.
+
+But let's have a look at the Cypher statement to create the relationships:
+
+.Load statement
+[source,cypher]
+----
+UNWIND $relRecords AS relRecord
+MATCH (source: `Order` { `orderID`: toInteger(trim(relRecord.`orderID`)) })
+MATCH (target: `Product` { `productID`: toInteger(trim(relRecord.`productID`)) })
+MERGE (source)-[r: `CONTAINS`]->(target);
+SET r.`unitPrice` = toFloat(trim(relRecord.`unitPrice`))
+SET r.`quantity` = toInteger(trim(relRecord.`quantity`))
+SET r.`discount` = toFloat(trim(relRecord.`discount`));
+----
+
+From this you can see how a relationship is created _from_ a source node _to_ a target node and how each created relationship is of the type `CONTAINS`.
```

Comment:
to be precise, the nodes at each end are MATCHed (found in the data) and then a MERGE statement is used to create (or update an existing) relationship between those two nodes.

Reply:
Yes, this is great! I've added!